docs/user/science.rst

```diff
@@ -0,0 +1,142 @@
+
+.. dropdown:: 📚 Bibliographies and external links
+
+    Maintain bibliography databases directly as code and have external links automatically verified.
+
+    Using extensions for Sphinx such as the popular `sphinxcontrib-bibtex <https://sphinxcontrib-bibtex.readthedocs.io/>`__ extension, you can maintain your bibliography with Sphinx directly or refer to entries ``.bib`` files, as well as generating entire Bibliography sections from those files.
+
+.. dropdown:: 📜 Modern themes and classic PDF outputs
+
+    .. figure:: /img/screenshot_rtd_downloads.png
+        :align: right
+
+    Use the latest state-of-the-art themes for web and have PDFs and e-book formats automatically generated.
+
+    New themes are improving every day, and when you write documentation based on Jupyter Book and Sphinx, you will separate your contents and semantics from your presentation logic. This way, you can keep up with the latest theme updates or try new themes.
+
+    Another example of the benefits from separating content and presentation logic: Your documentation also transforms into printable books and eBooks.
+
+.. dropdown:: 📐 Widgets, widgets and more widgets
+
+    Design your science project's layout and components with widgets from a rich eco-system of Open Source extensions built for many purposes. Special widgets help users display and interact with graphs, maps and more. :external+jupyterbook:doc:`Several <content/components>` `extensions <https://sphinx-gallery.github.io/>`__ are built and invented by the science community.
+
+.. dropdown:: ⚙️ Automatic builds
+
+    Build and publish your project for every change made through Git (GitHub, GitLab, BitBucket etc). Preview changes via Pull Requests. Receive notifications when something is wrong. How does this work? Have a look at this video:
+
+    .. TODO: Where should this video live?
+
+    .. video:: https://overtag.dk/files/enable-pull-request-builders.mp4
+       :width: 500
+       :height: 300
+
+.. dropdown:: 💬 Collaboration and community
+
+    .. figure:: /img/screenshot_edit_on_github.png
+        :align: right
+
+    Science and academia have a big kinship with software developers: We ❤️ community. Our solutions and projects become better when we foster inclusivity and active participation. Read the Docs features easy access for readers to suggest changes via GitHub, GitLab, BitBucket etc. But not just any unqualified feedback. Instead, the *code* and all the tools are available for your community to forge qualified contributions.
+
+    Your readers can become your co-authors!
+
+    Discuss changes via Pull Request and track all changes in your project's version history.
+
+    Does this sound chaotic? Don't worry, the full ownership and permission handling remains in your hands: Project and organization owners govern what is released and who has access to approve and build changes.
+
+.. dropdown:: 🔎 Search Engine Optimization
```

Comment:
This section feels a little out of place, perhaps it could be generalized to cover more search features instead?

Neither RTD or Sphinx have really strong features to help SEO, but we do have strong search features otherwise. Maybe what we're focusing on here is that readers can easily find what they are looking for, and enumerate search features instead.

Unless we have input from projects that list SEO as a strong concern for their project, though search just in general is usually a higher priority item for projects.